content/learning/operator/operator.md

```diff
@@ -69,6 +69,54 @@ The kubebuilder framework uses the controller-runtime library to communicate wit
 4. the [request](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/reconcile#Request "request") received in the reconciler function
 5. k8s [corev1](https://godoc.org/k8s.io/api/core/v1 "corev1") in case you need to interact with core k8s kind (pods, deployments, configmaps etc)
 
+## Useful tips
+
+Here are some useful suggestion if you are new to programming an operator:
+1. You may need to watch a second resource. In this case what you need to do is to add Owns in the [setup](https://book.kubebuilder.io/cronjob-tutorial/controller-implementation.html#setup) function :
+```
+func (r *_ObjectToReconcile_) SetupWithManager(mgr ctrl.Manager) error {
+	
+	return ctrl.NewControllerManagedBy(mgr).
+		For(&principal_resource).
+		Owns(&second resource_to_watch).
+		Complete(r)
+}
+```
+
+2. It is possible to filter the case of reconcile that our operator perform thanks to the [Predicates](https://sdk.operatorframework.io/docs/building-operators/golang/references/event-filtering/). To use them you can modify the setup function explayned above:
```

Comment:
* Please, give an example why you would do this and explain the example code. Remember to put yourself in the shoes of a newbie reader that knows nothing about how to build an operator, not to mention the advanced aspects. Would you understand? Keep iterating and improving the documentation until the answer is yes.
* *filter the case of reconcile* -> what do you mean? Absolutely not clear to me
* explayned -> explained